src/components/Share/Share.css

```diff
@@ -0,0 +1,60 @@
+.Share {
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: 0;
+  margin: 0.5rem;
+  line-height: 1.5rem;
+
+  h1 {
+    display: inline-block;
+    font-size: 1.5rem;
+    font-weight: 100;
+
+    strong {
+      font-weight: 600;
+    }
+  }
+
+  input {
+    position: absolute;
+    left: -99999px;
```

Comment:
I'll try those ways we discussed about, but frankly, they looks pretty much the same to me. 😉